docs/docs/recipes/gitlab-continuous-integration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Recipes: Continuous Integration on Gitlab"
+tableOfContentsDepth: 1
+---
+
+Push your code to gitlab and automate your production build!
+
+### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
```

Comment:
We prefer to minimise the use of external links in doc to keep the reader focused on subject matter. Can we replay this step with just having the CLI installed,  then have setting up a Gatsby site as step 1 in directions. 
```suggestion
- Make sure you have the [Gatsby CLI](docs/gatsby-cli) installed
```